.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+php: 
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
```

Comment:
přidat nightly a u něj "can fail". Ať víme, že nám to padá na 7.4, atp.

Reply:
* https://github.com/keboola/php-component/blob/master/.travis.yml#L9
* https://github.com/keboola/php-component/blob/master/.travis.yml#L20
* https://github.com/keboola/php-component/blob/master/.travis.yml#L12
* https://github.com/keboola/php-component/blob/master/.travis.yml#L24

Reply:
nightly jsem pridal, ale matrix tam nema smysl, kdyz to nema zadny composer zavislosti ne?

Reply:
Jo, pravda